dashboard/test/controllers/aichat_events_controller_test.rb

```diff
@@ -113,36 +113,36 @@ class AichatEventsControllerTest < ActionController::TestCase
   end
 
   # *****
-  # student_chat_history tests
+  # chat_history tests
   # *****
 
-  test 'Bad request if required params are not included for student_chat_history' do
+  test 'Bad request if required params are not included for chat_history' do
     sign_in(@authorized_teacher1)
-    get :student_chat_history, params: {studentId: @authorized_student1.id}, as: :json
+    get :chat_history, params: {studentId: @authorized_student1.id}, as: :json
     assert_response :bad_request
   end
 
-  test 'student of authorized teacher does not have access to student_chat_history test' do
+  test 'student of authorized teacher has access to chat_history' do
```

Comment:
Can we add a test to ensure that a student cannot access another student's chat history?

Reply:
Yep, added!